### DISCUSSION TITLE
The display of the status of pull request on issue timeline seems confusing, especially for Github beginners

### DISCUSSION THREAD

**Discussion by 3-2** [community/community#17321] [Projects and Issues]
In issue timeline, there is a status label besides the pull request items. It shows 'Merged' if the pull request was merged.

EXAMPLE
<img width="896" alt="image" src="https://user-images.githubusercontent.com/25643218/170535074-7ffdab63-1498-4448-8662-679a8fab5966.png">
(Source: https://github.com/AdguardTeam/AdguardFilters/issues/119683)

PROBLEM
I am a Github beginner.
At the moment I saw this pull request item on the timeline, I thought 'Oh, there is a pull request and has been merged, but why are there more timeline items (some commits) after this merged pull request?'
Now I understood what's going on. I think it's worth improving such confusing thing for beginners.

SUGGESTION
Display merged time inside or besides the label except for just the word 'Merged'

**Comment by mfsbo** [comment]
I think you are looking at timeline event of mention and not the event merge. The person at that time mentioned the PR and if you went to see the timeline at between merge and mention the PR might have gone from Open to Draft or Close so the status is shown as its current status and not when it was merged.
Having another date/time on this timeline event will confuse people with the time of mentione event and time of merge event.
